src/main/resources/init_db.sql

```diff
@@ -0,0 +1,3 @@
+CREATE DATABASE book;
+USE book;
+CREATE TABLE books ( id BIGINT PRIMARY KEY, title VARCHAR(255), price INT );
```

Comment:
I think you need to format the sql query to create the table, and also the id should probably be AUTO_INCREMENT.
```suggestion
CREATE TABLE books
(
    id    BIGINT PRIMARY KEY AUTO_INCREMENT,
    title VARCHAR(255),
    price INT
);
```